portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,25 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getHello() {
+    fetch('/data')
+    .then(response => response.json())
+    .then((funFacts) => {
+        const uname  = document.getElementById("username");
```

Comment:
Super nitpick: Use consistent quotes, either all single or all double.

Reply:
Interesting, I didn't even notice I used double quotations instead of single quotations. Later on in the code, I call document.getElementById but use single quotations. Does JavaScript not care about which ones I use?

Reply:
Also, I made the changes so I'm only using single quotations.